cartoframes/io/carto.py

```diff
@@ -61,7 +61,7 @@ def read_carto(source, credentials=None, limit=None, retry_times=3, schema=None,
 
 
 def to_carto(dataframe, table_name, credentials=None, if_exists='fail', geom_col=None, index=False, index_label=None,
-             log_enabled=True, force_cartodbfy=False):
+             force_cartodbfy=False, log_enabled=True):
```

Comment:
I would choose the log level and I would remove this  `log_enabled` parameter in this method and the rest

Reply:
This is required because "carto methods" are used internally, for example in the geocoding, so if this is the case, we want to show only the geocoding message, instead of the `"Success! Data uploaded" message` of the intermediate step. Both are "info" level logs.